src/oarepo_model/presets/relations/ext.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2025 CESNET z.s.p.o.
+#
+# This file is a part of oarepo-model (see http://github.com/oarepo/oarepo-model).
+#
+# oarepo-model is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+"""A module for defining presets for model relations."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, override
+
+import oarepo_model
+from oarepo_model.customizations import (
+    AddMixins,
+    Customization,
+)
+from oarepo_model.presets import Preset
+
+if TYPE_CHECKING:
+    from collections.abc import Generator
+
+    from oarepo_model.builder import InvenioModelBuilder
+    from oarepo_model.model import InvenioModel
+
+
+class RelationsFeaturePreset(Preset):
+    """Preset for enabling relations feature."""
+
+    @override
+    def apply(
+        self,
+        builder: InvenioModelBuilder,
+        model: InvenioModel,
+        dependencies: dict[str, Any],
+    ) -> Generator[Customization]:
+        class RelationsFeatureMixin:
+            @property
+            def model_arguments(self) -> dict[str, Any]:
+                """Model arguments for the extension."""
+                parent_model_args = super().model_arguments  # type: ignore[misc] # pyright: ignore[reportAttributeAccessIssue]
+                return {
+                    **parent_model_args,
+                    "features": {**parent_model_args["features"], "relations": {"version": oarepo_model.__version__}},
```

Comment:
note to self: what version should be here? oarepo_model or invenio_records_resources?